### DISCUSSION TITLE
Ctrl-Enter  &  Shift-Ctrl-Enter  not working

### DISCUSSION THREAD

**Discussion by HaleTom** [wezterm/wezterm#5774] [General]
Hi all,

I'm trying to get `Ctrl-Enter` and `Shift-Ctrl-Enter` to send different codes to plain `Enter`.

Passing them through `cat | xxd`, both `Ctrl-Enter` & `Shift-Ctrl-Enter` give `0a` (the same as plain `Enter`)

How do I get different codes when the modifiers are pressed?

(I'm a Manjaro user, if it makes any difference)

**Comment by meepleek** [comment]
Enabling kitty kb encoding should help with this.  
https://wezfurlong.org/wezterm/config/lua/config/enable_kitty_keyboard.html  
https://wezfurlong.org/wezterm/config/key-encoding.html#kitty-keyboard-protocol

**Comment by cstrahan** [comment]
I'll second the advice to use [`enable_kitty_keyboard`](https://wezfurlong.org/wezterm/config/lua/config/enable_kitty_keyboard.html
).

Using `kitten show_key -m kitty` to verify (`kitten` comes from the [kitty terminal emulator](https://sw.kovidgoyal.net/kitty/)), I see that `Ctrl-Enter` and `Shift-Ctrl-Enter` are encoded as expected:

```
ctrl+ENTER PRESS
CSI 13 ; 5 ; 13 u

ctrl+ENTER RELEASE
CSI 13 ; 5 : 3 u

shift+ctrl+ENTER PRESS
CSI 13 ; 6 ; 13 u

shift+ctrl+ENTER RELEASE
CSI 13 ; 6 : 3 u
```
